index.rst

```diff
@@ -14,35 +14,26 @@ Quick Start
 Here are the basic steps needed to get :ref:`set up <setup>` and contribute a
 patch:
 
-1. Set up and install dependencies.
+1. Install and set up :ref:`Git <vcsetup>` and other dependencies
+   (see the :ref:`Get Setup <setup>` page for detailed information).
 
-   Install :ref:`Git <vcsetup>` and other dependencies.
+2. Fork `the CPython repository on GitHub <cpython github>`_ and
```

Comment:
@ezio-melotti For someone that has used git, your wording is fine. If someone new to git/GitHub is trying to help, they will often get confused about what is getting forked and cloned where. I know it seems trivial, but it really does help those who are still learning.